src/shared/store/actions/nexus/index.ts

```diff
@@ -161,7 +302,22 @@ export const fetchResources: ActionCreator<ThunkAction> = (
     try {
       const org: Organization = await nexus.getOrganization(orgLabel);
       const project: Project = await org.getProject(projectLabel);
-      const resources: PaginatedList<Resource> = await project.listResources(
+      const elasticSearchViews: ElasticSearchView[] = await project.listElasticSearchViews();
+      const defaultElasticSearchView: ElasticSearchView = elasticSearchViews[0];
```

Comment:
Not sure the first one in the list will always be the default one. Shouldn't we add getDefaultElasticSearchView() to the sdk and discriminate based on '@id'?

Reply:
Thanks for adding this feature to the sdK!